### PR TITLE
Tg/adding step

### PR DESCRIPTION
**Pull request by Bagra67**

### CODE REVIEW

File: src/data/points/points.ts [hunk 1]
```diff
@@ -67,7 +67,7 @@ export async function getStepPoints(stepId: number): Promise<PointOutput[]> {
 export async function updatePoint(stepId: number, pointId: number, data: Partial<PointInput>): Promise<PointOutput> {
 
 	try {
-		const response = (await request(`/trips/${stepId}/points${pointId}`, "PUT", data));
+		const response = (await request(`/steps/${stepId}/points${pointId}`, "PUT", data));
```
Comment (Kayn017) [inline src/data/points/points.ts:70]: Not the right endpoint
Delete the parameter "stepId" et change endpoint to "/points/:pointId"

https://github.com/La-Sectoblique/septotrip-api/blob/36cc241fc79522a90f378c89d959403b7eec1171/src/routes/protected/points.ts#L15

File: src/data/points/points.ts [hunk 2]
```diff
@@ -100,7 +100,7 @@ export async function updatePoint(stepId: number, pointId: number, data: Partial
 export async function deletePoint(stepId: number, pointId: number) {
 
 	try {
-		const response = (await request(`/trips/${stepId}/points${pointId}`, "DELETE"));
+		const response = (await request(`/steps/${stepId}/points${pointId}`, "DELETE"));
```
Comment (Kayn017) [inline src/data/points/points.ts:103]: Same

Comment (Bagra67) [reply src/data/points/points.ts]: done

File: src/data/points/points.ts (hunk 1 above)
Comment (Bagra67) [reply src/data/points/points.ts]: done

File: src/data/points/points.ts [hunk 3]
```diff
@@ -59,15 +59,14 @@ export async function getStepPoints(stepId: number): Promise<PointOutput[]> {
...
  * @param pointId identifiant du point
  * @param data nouvelles données
  * @returns le point modifié
  */
-export async function updatePoint(stepId: number, pointId: number, data: Partial<PointInput>): Promise<PointOutput> {
+export async function updatePoint(pointId: number, data: Partial<PointInput>): Promise<PointOutput> {
 
 	try {
-		const response = (await request(`/steps/${stepId}/points${pointId}`, "PUT", data));
+		const response = (await request(`/points${pointId}`, "PUT", data));
```
Comment (Kayn017) [inline src/data/points/points.ts:69]:
```suggestion
		const response = (await request(`/points/${pointId}`, "PUT", data));
```

File: src/data/points/points.ts [hunk 4]
```diff
@@ -93,14 +92,13 @@ export async function updatePoint(stepId: number, pointId: number, data: Partial
...
- * @param stepId identifiant de l'étape
  * @param pointId identifiant du point
  * @returns 
  */
-export async function deletePoint(stepId: number, pointId: number) {
+export async function deletePoint(pointId: number) {
 
 	try {
-		const response = (await request(`/steps/${stepId}/points${pointId}`, "DELETE"));
+		const response = (await request(`/points${pointId}`, "DELETE"));
```
Comment (Kayn017) [inline src/data/points/points.ts:101]:
```suggestion
		const response = (await request(`/points/${pointId}`, "DELETE"));
```

**Pull request merged**